helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -365,7 +365,6 @@ To configure this, set the following environment variables on
 1. The deployments based on the `fiftyone-teams-app` (`teamsAppSettings.env`) or
    `fiftyone-teams-cas` (`casSettings.env`) images
 
-
```

Comment:
Removed extra unnecessary new lines that showed as warnings for markdownlint plugin